examples/resources/maas_boot_source/import.sh

```diff
@@ -0,0 +1,2 @@
+# Devices can be imported using their ID, e.g:
```

Comment:
Import is not valid for this resource, since Terraform assumes that there is only one boot source. Please remove it